src/RageSoundPosMap.cpp

```diff
@@ -176,20 +176,15 @@ int64_t pos_map_queue::Search( int64_t iSourceFrame, bool *bApproximate ) const
 	 *    SoundStopped has been called.
 	 * 3. Underflow; we'll be given a larger frame number than we know about.
 	 */
-#if defined(WIN32)
```

Comment:
I assume these are dead defines?

Reply:
They were used but certain compilers did not like how they were used in the singular `printf`-style log statement below. I changed the code to use `std::stringstream` to format the numbers to avoid the problem altogether.

Reply:
Ahh gotcha. Sounds good.